chef/cookbooks/nova_dashboard/attributes/default.rb

```diff
@@ -23,6 +23,15 @@
 default[:nova_dashboard][:site_branding_link] = ""
 default[:nova_dashboard][:help_url] = "http://docs.openstack.org/"
 
+default[:nova_dashboard][:policy_file_path] = nil
```

Comment:
I was using `.to_s.empty?` Somehow I used the password example, but I will change it.